daemon/cmd/cmdhelp_template.go

```diff
@@ -1,98 +0,0 @@
-// Copyright 2020 Authors of Cilium
```

Comment:
Regarding the help changes:

> There are various issues, e.g. flags are forgotten to be added to sections given this needs to be explicitly specified and then we end up with a random chunk of help
flags at the end of the --help dump.

If the random chunk at the end is the problem, I think we can write some simple trickery to ensure the help commands are categorized as part of the build process, so if they're not categorized then we tell the contributor to go back and categorize the flag.

Personally I don't really refer to the help text at all so I can't really speak for whether the sectioning is useful or not, but I was a little surprised to see a CLI change like this in an LB followups PR. Has there been a community discussion on this topic?

The change itself seems to accurately reflect the intent though, so code-wise :+1:

Reply:
I've moved the commit out and will open a separate PR for dicussion.